modules/add-contextual-data/tests/test_context_info_db.c

```diff
@@ -194,15 +195,15 @@ Test(add_contextual_data, test_inserted_nv_pairs)
   ContextInfoDB *context_info_db = context_info_db_new();
   _fill_context_info_db(context_info_db, "selector", "name", "value", 1, 3);
 
-  TestNVPair expected_nvpairs[3] =
+  TestNVPair expected_nvpairs[] =
```

Comment:
Using 'hard-coded' values in the testcase was intentional.
I don't mind it, but I don't think this makes much more readable the code: in an assert I prefer concrete values instead of computed values.